projects/nodejs/sydr_main.cc

```diff
@@ -0,0 +1,24 @@
+#include <cstdint>
```

Comment:
Now `/opt/StandaloneFuzzTargetMain.c` is in docker.

Reply:
I need "extern \"C\"" in  `/opt/StandaloneFuzzTargetMain.c`  to compile target

Reply:
reworked: building StandaloneFuzzTargetMain.c into object file without patch